From: SeongJae Park <sj@kernel.org>
Date: Sun, 27 Oct 2024 13:34:49 -0700
Subject: [PATCH] Fix temporal build error

Build fails on CONFIG_MEMCG=n configs.  Add the suggested fix until it
is merged into mm-unstable.

Link: https://lore.kernel.org/20241027173338.148147-1-sj@kernel.org
Signed-off-by: Shakeel Butt <shakeel.butt@linux.dev>
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 include/linux/memcontrol.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/include/linux/memcontrol.h b/include/linux/memcontrol.h
index 89a1e9f10e1b..5502aa8e138e 100644
--- a/include/linux/memcontrol.h
+++ b/include/linux/memcontrol.h
@@ -1055,6 +1055,11 @@ static inline struct mem_cgroup *folio_memcg(struct folio *folio)
 	return NULL;
 }
 
+static inline bool folio_memcg_charged(struct folio *folio)
+{
+	return false;
+}
+
 static inline struct mem_cgroup *folio_memcg_check(struct folio *folio)
 {
 	return NULL;
-- 
2.39.5

